demos/multi_channel/object_detection_demo_yolov3/multichannel_object_detection_demo_yolov3_params.hpp

```diff
@@ -14,3 +14,88 @@ static const char thresh_output_message[] = "Optional. Probability threshold for
 /// \brief Flag to output raw scoring results<br>
 /// It is an optional parameter. Ignored for human-pose-estimation
 DEFINE_double(t, 0.5, thresh_output_message);
+
+std::vector<cv::Scalar> colors(80); 
```

Comment:
This is not an option, so it shouldn't be in this file. In fact, it shouldn't be a global at all - it should just be a local variable in `main`.